### PR TITLE
RAB-1300: Define a limit for attribute groups creation

### PR DESCRIPTION
**Pull request by ghoussard**
**Description (for Contributor and Core Developer)**

In this PR, 

We added a limit of attribute groups, now user cannot anymore create an attribute group when it reach 1000 attribute groups. It can continue to edit attribute group even if he as more than 1000 attribute groups.

Actually only one client have more than 1000 attribute groups (half have no attribute inside). The client have been advert. We didn't handle correctly the import due to the fact that attribute group are created by batch (unitary validated before). We cannot create more that 1000 + attribute group batch size (100)

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/attribute_group/index.yml [hunk 1]
```diff
@@ -1,47 +0,0 @@
-extensions:
```
Comment (ghoussard) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/attribute_group/index.yml:1]: Not used anymore

**Review by StevenVAIDIE — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/attribute_group/index.yml (hunk 1 above)
Comment (ValentinMumble) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/config/form_extensions/attribute_group/index.yml:1]: 🔥

**Review by ValentinMumble — APPROVED** [review]

**Review by mmetayer — COMMENTED** [review]
some tests are missing:
- unit tests for the new validator
- being able to update an existing attribute group even if the limit is reached
- what happens for import or internal API? (best way to test that would be an acceptance test)

**Review by pierallard — APPROVED** [review]
We usually don't put const in Validator (your limit for example), but as a parameter in the .yml validator.
But that's OK for me, it's just a detail.

**Review by mmetayer — APPROVED** [review]

**Pull request merged**